test/parallel/test-fs-rmdir-recursive.js

```diff
@@ -10,17 +10,41 @@ let count = 0;
 
 tmpdir.refresh();
 
-function makeNonEmptyDirectory() {
-  const dirname = `rmdir-recursive-${count}`;
-  fs.mkdirSync(path.join(dirname, 'foo', 'bar', 'baz'), { recursive: true });
+function makeNonEmptyDirectory (depth, files, folders, dirname) {
+  fs.mkdirSync(dirname, { recursive: true });
   fs.writeFileSync(path.join(dirname, 'text.txt'), 'hello', 'utf8');
-  count++;
-  return dirname;
+
+  var o = { flag: 'wx' }
+  if (process.version.match(/^v0\.8/))
+    o = 'utf8'
+
+  for (var f = files; f > 0; f--) {
+    fs.writeFileSync(dirname + '/f-' + depth + '-' + f, '', o)
```

Comment:
I think we can use template literals to build these file names and joining the dirname and filename should probably use `path.join` instead of hardcoding `/`.